app/poc-common/clinical-services/directives/clinicalServices.html

```diff
@@ -39,7 +39,7 @@
         <div class="col-sm-2 go-to-right">
 
           <poc-authorize privilege="vm.getPrivilege('Write', service)">
-            <button ng-if="vm.canAdd(service)" class="btn btn-primary btn-lg" type="button"
+            <button ng-if="vm.patientCheckedIn && vm.canAdd(service)" class="btn btn-primary btn-lg" type="button"
                     ng-click="vm.linkServiceAdd(service)">
```

Comment:
Should be able to add **Vitals** without checkin. **Vitals** is another mechanism for checkin.
![image](https://user-images.githubusercontent.com/3810265/36663476-5d82ad24-1aea-11e8-9918-25010bade45a.png)